template.yaml

```diff
@@ -79,7 +79,10 @@ Mappings:
       docAppStubUrl: https://doc-app-rp-staging.build.stubs.account.gov.uk
       testClientDefaultClientId: nsR2wZ7EebJ2VOzE1LUa9iAVadunWQP3
       testClientStubUrl: https://perf-test-rp-staging.build.stubs.account.gov.uk
-      testClientInstances: 6
+      testClientInstances: 4
+      testClientInstancesmincount: 4
+      testClientInstancesmaxcount: 30
+      ALBPolicyTargetValue: 2000
```

Comment:
We also need these in the build mapping, right?

Reply:
Added the value for Build env Also